easybuild/easyconfigs/p/PSI4/PSI4-1.0-intel-2016a-mt-Python-2.7.11.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'EB_PSI'
+
+name = 'PSI4'
+version = '1.0'
+versionsuffix = '-mt'
+
+homepage = 'http://www.psicode.org/'
+description = """PSI4 is an open-source suite of ab initio quantum chemistry programs designed for
+ efficient, high-accuracy simulations of a variety of molecular properties. We can routinely perform
+ computations with more than 2500 basis functions running serially or in parallel."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'usempi': False}
+
+source_urls = ['https://github.com/psi4/psi4/archive/']
+sources = ['%(version)s.tar.gz']
+
+patches = [
+    'PSI4-plugin-fix.patch',
+    'PSI4-1.0-disable-ambit.patch',
+]
+
+pysuff = '-Python-%(pyver)s'
+versionsuffix += pysuff
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('Boost', '1.61.0', pysuff),
+    ('PCMSolver', '1.1.4', pysuff),
+    ('CheMPS2', '1.7.1'),
+]
+
+builddependencies = [
+    ('CMake', '3.5.2'),
+    ('Perl', '5.22.1', '-bare'),  # for the test suite
+]
+
+configopts = '-DENABLE_MPI=OFF -DENABLE_OMP=ON -DENABLE_UNIT_TESTS=ON -DENABLE_CHEMPS2=ON -DENABLE_PLUGINS=ON'
+configopts += ' -DENABLE_DUMMY_PLUGIN=ON -DCHEMPS2_ROOT=$EBROOTCHEMPS2'
+configopts += ' -DENABLE_PCMSOLVER=ON -DPCMSOLVER_ROOT=$EBROOTPCMSOLVER'
```

Comment:
shouldn't some of these be tackled in the PSI easyblock?

if `CheMPS2` is included as a dep, use `-DENABLE_CHEMPS2=ON -DCHEMPS2_ROOT=$EBROOTCHEMPS2`, etc.